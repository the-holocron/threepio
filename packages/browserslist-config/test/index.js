/* eslint-disable no-console */
const browserslist = require("browserslist");
const config = require("../");

const browsers = browserslist(config);

describe("configuration", () => {
	describe("browserslist", () => {
		test("should return a list of browsers", () => expect(browsers.length).toBeGreaterThan(0));
		test("should render the configuration", () => expect(browsers).toMatchSnapshot());
		afterAll(() => console.log("Resolved the following browsers:\n\n", browsers.join("\n")));
	});
});
