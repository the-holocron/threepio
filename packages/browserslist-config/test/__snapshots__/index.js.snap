// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`configuration browserslist should render the configuration 1`] = `
Array [
  "and_chr 120",
  "chrome 121",
  "chrome 120",
  "chrome 119",
  "edge 121",
  "edge 120",
  "firefox 122",
  "firefox 121",
  "ios_saf 17.1",
  "safari 17.3",
  "safari 17.2",
]
`;
